tools/bin/gppylib/commands/unix.py

```diff
@@ -925,7 +925,7 @@ def getDescendentProcesses(pid):
     children = list()
     grandchildren = list()
 
-    for p in psi.process.ProcessTable().values():
+    for p in psutil.process.ProcessTable().values():
```

Comment:
@cbaenziger This usage here is wrong since `psutil` module has no attribute `process`. Why should you use `psutil` in stead of `psi`?